### ISSUE TITLE
Missing server version and name

### ISSUE THREAD

**Issue by treemlt** [xstar97/palworld-query-api#1]
{"default":{"serverName":"","serverVer":"","players":{"count":5,"list":["hata","ahraj","fune","dire Fae","veed"]}}}

This is the output i am getting. It's missing serverVer and server name

Thanks

**Comment by xstar97** [comment]
Logs that i have set wont be much help to diagnose the issue here, do you use special chars in your server name and would you be able to share a variant of the name so i can test it for validation purposes.

**Comment by treemlt** [comment]
This is the server name **TrEE`Gaming PalWorld [EU][24/7]** but also the version is not being displayed

**Comment by treemlt** [comment]
```
Welcome to Pal Server[v0.1.5.1] TrEE`Gaming PalWorld [EU][24/7]

[2024-03-06 17:08:14] 192.168.178.38:25575: showplayers
nme,playeruid,steamid
Sta,502139665,76561198032962925
Traj,3723788180,76561198832419042
Tne,1092809199,76561198019443589
Fie Fae,2105230841,76561128084073410
Red,1826119273,76561191956552414
```


In the logs it shows all the data but when using the api it doesn't

**Comment by xstar97** [comment]
The issue is with special characters I'll have a fixed dropped soon,  it will fix both the name and the version.

**Comment by xstar97** [comment]
fixed https://github.com/xstar97/palworld-query-api/releases/tag/v0.0.2

https://github.com/xstar97/palworld-query-api/pkgs/container/palworld-query-api/187804984?tag=v0.0.2

**Comment by xstar97** [comment]
should be fixed in v0.0.5 now released.
